pkg/networkmanager/networking.css

```diff
@@ -198,4 +198,16 @@ h1 .btn-onoff-ct {
 #add-services-dialog .list-group-item input[type="checkbox"] {
     margin: 0;
     vertical-align: middle;
+
+}
+.toggle-body > input {
+    margin-right: 2px;
+}
+
+.toggle-body > :not(input) {
+    margin-left: 14px;
+}
+
+.form-control.error {
+    border-color: red;
```

Comment:
Why not use [native form validation](https://developer.mozilla.org/en-US/docs/Learn/HTML/Forms/Form_validation#The_required_attribute)?

Reply:
> Why not use native form validation?

Because the checking is rather complex and firstly needs loading `/etc/services`

> Alternatively, there's a .has-error class in PatternFly

awesome, that is what I was looking for :)

Reply:
> Alternatively, there's a .has-error class in PatternFly, but that needs to be applied to a parent element.

The fact it needs to be applied to a parent element is rather impractical - I need to put \<input\> into \<div\> or other element. We define out own `.has-error` in `pkg/systemd/timer.css`. Would that be a solution?